tiledb/libtiledb.pyx

```diff
@@ -1275,7 +1275,10 @@ cdef _numpy_dtype(tiledb_datatype_t tiledb_dtype, cell_size = 1):
     cdef base_dtype
     cdef uint32_t cell_val_num = cell_size
 
-    if cell_val_num == 1:
+    if tiledb_dtype == TILEDB_BLOB:
```

Comment:
Can we pull the BLOB types out to another PR so we can have compatibility array created with this type?